modules/modules.nix

```
@@ -215,6 +215,7 @@ let
     (loadModule ./services/wlsunset.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/xcape.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/xembed-sni-proxy.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./services/xidlehook.nix { })
```

```suggestion
    (loadModule ./services/xidlehook.nix { condition = hostPlatform.isLinux; })
```

I don't think xidlehook works with Quartz or other X11 servers on other platforms.